### PR TITLE
{bio}[GCC/6.4.0-2.28,iccifort/2017.4.196-GCC-6.4.0-2.28] PEAR v0.9.10

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/9e510ba014d8ef60648423a9adcadf74 for a full test report.

**Comment by boegel** [comment]
@vanzod Hmm, auto-download doesn't work with specified source URL?

**Comment by vanzod** [comment]
@boegel Initially I thought the website was under maintenance. Then I have actually found out that now you have to request the software on a different URL and that only the latest version is available.
I have added a comment in the easyconfig file.

File: easybuild/easyconfigs/p/PEAR/PEAR-0.9.10-iccifort-2017.4.196-GCC-6.4.0-2.28.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+homepage = 'http://sco.h-its.org/exelixis/web/software/pear/'
+description = """PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger. 
+ It is fully parallelized and can run with as low as just a few kilobytes of memory."""
+
+toolchain = {'name': 'iccifort', 'version': '2017.4.196-GCC-6.4.0-2.28'}
+
+# The source archive must now be requested at
+# https://www.h-its.org/en/research/sco/software/#NextGenerationSequencingSequenceAnalysis
+
+sources = ['%(namelower)s-%(version)s.tar.gz']
```
Comment (boegel) [inline easybuild/easyconfigs/p/PEAR/PEAR-0.9.10-iccifort-2017.4.196-GCC-6.4.0-2.28.eb]: @vanzod Should be `'%(namelower)s-src-%(version)s.tar.gz'`?

Not sure how the test worked for you with this in place...

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/04a88d923731d29d2268b152c0be0b3c for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/5e45bbbc6082a272a0b59363fb3438c3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
gligar03.gligar.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/dc2082d083fb4e2632e4cf9d24921751 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2482.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/dabf9914ad5cb5de3b770cdc0de23790 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**